travis_specific/tests.sh

```diff
@@ -1,5 +1,13 @@
 #!/bin/bash
```

Comment:
Now would be a great time to rename this file to fabric_tests.sh.

Reply:
yes.